mpop_kiosk/PageGender.qml

```diff
@@ -11,10 +11,17 @@ RowLayout {
     signal nextButtonClicked()
     signal previousButtonClicked()
 
-    Label {
-        Layout.alignment: Qt.AlignCenter
-        text: qsTr("You are...")
-        font.pixelSize: 36
+    Rectangle{
+        Layout.minimumWidth: 400
+        Layout.minimumHeight: 500
+        color: "black"
+
+        Label {
+            Layout.alignment: Qt.AlignCenter
+            text: qsTr("VOUS ETES...")
```

Comment:
Here, instead of `qsTr("VOUS ETES...")` I would go `qsTr("You are...")` and then add `font.capitalization: Font.AllUppercase`